crates/matrix-sdk/src/pusher.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2024 FIXME help with copyright wording
```

Comment:
Since there's just a tiny bit of code motion, we need to include the previous copyright attribution here, and since you've added code, you can put your own attribution too. So something like:

```suggestion
// Copyright 2024 The Matrix.org Foundation C.I.C.
// Copyright 2024 hanadi92 // or your first+last names if you want, or your favorite nickname
```